README.md

```diff
@@ -92,6 +92,45 @@ Each of the timers has the following keys:
 
 The state value shows the next timer as a timestring (i.e.: `2021-03-07T15:26:17+01:00`) if there is at least one timer set, otherwise it is set to `unavailable`.
 
+### Services
```

Comment:
I think `Features` section is pretty long already, and it's better to add subsections to the contents menu for easier navigation.